tests/mocks/HUBComponentMock.h

```diff
@@ -91,6 +91,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Whether the component should act like it is a content offset observer or not
 @property (nonatomic) BOOL isContentOffsetObserver;
 
+/// Whether the component's image was recently animated
+@property (nonatomic) BOOL isImageAnimated;
```

Comment:
Also since it refers to the past, how about `imageWasAnimated`? (just a suggestion)